connect_telegram_ext/messages.py

```diff
@@ -19,13 +19,13 @@
 # Copyright (c) 2022, Cloudblue Connect
 # All rights reserved.
 #
-DEFAULT_MESSAGE = 'Obtained request №[{id}]({object_link}) in status {object_status}'
+DEFAULT_MESSAGE = 'Obtained request №<a href={object_link}>{id}</a> in status {object_status}'
 
 ASSET_ADJUSTMENT_REQUEST_PROCESSING = None
 ASSET_CANCEL_REQUEST_PROCESSING = None
 ASSET_CHANGE_REQUEST_PROCESSING = None
 ASSET_PURCHASE_REQUEST_PROCESSING = (
-    'Purchase request №[{id}]({object_link}) got update. '
+    'Purchase request №<a href={object_link}>{id}</a> got update. '
```

Comment:
Attention: Purchase request №[{id}](https://github.com/cloudblue/connect-extension-telegram-notifications/pull/3/%7Bobject_link%7D) has been updated and is in status {object_status}. Please check if requires your attention.